app/controllers/nonsipp/membercontributions/MemberContributionListController.scala

```diff
@@ -343,26 +255,22 @@ object MemberContributionListController {
       }
     )
 
-    val normalModeMessage =
-      if (mode == NormalMode)
-        Option(
-          ParagraphMessage(
-            "ReportContribution.MemberList.paragraph1"
-          ) ++
-            ParagraphMessage(
-              "ReportContribution.MemberList.paragraph2"
-            )
+    val optDescription = Option.when(mode == NormalMode)(
```

Comment:
Is this now changes the behaviour?
I..e previously it evaluated to `Option(ParagraphMessage("")` if not NormalMode, now it evaluates to `None`

Reply:
https://github.com/hmrc/pension-scheme-return-frontend/blob/main/app/views/TwoColumnsTripleAction.scala.html#L106

I didn't see any behaviour change on my local. As it's optional, it's cleaner to send None instead of Empty String if not NormalMode unless I miss any other aspects.

Reply:
Thanks @cyeniceri !  
@Vamshi-Muniganti could you please double-check if these descriptions are still working as expected when testing this ticket?